new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioMiiDomain.java

```diff
@@ -0,0 +1,297 @@
+import static oracle.weblogic.kubernetes.utils.IstioUtils.deployIstioDestinationRule;
+import static oracle.weblogic.kubernetes.utils.IstioUtils.getIstioHttpIngressPort;
+import static oracle.weblogic.kubernetes.utils.ThreadSafeLogger.getLogger;
+import static org.awaitility.Awaitility.with;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@DisplayName("Test istio enabled WebLogic Domain in mii model")
+@IntegrationTest
+class ItIstioMiiDomain {
+
+  private static HelmParams opHelmParams = null;
+  private static V1ServiceAccount serviceAccount = null;
+  private String serviceAccountName = null;
+  private static String opNamespace = null;
+  private static String operatorImage = null;
+  private static String domainNamespace = null;
+  private static ConditionFactory withStandardRetryPolicy = null;
+  private static ConditionFactory withQuickRetryPolicy = null;
+  private static String dockerConfigJson = "";
+
+  private String domainUid = "istio-mii";
+  private final String clusterName = "cluster-1"; // do not modify 
+  private final String adminServerName = "admin-server"; // do not modify
+  private String miiImage = null;
+  private static LoggingFacade logger = null;
+
+  private static Map<String, Object> secretNameMap;
+
+  /**
+   * Install Operator.
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+    // create standard, reusable retry/backoff policy
+    withStandardRetryPolicy = with().pollDelay(2, SECONDS)
+        .and().with().pollInterval(10, SECONDS)
+        .atMost(6, MINUTES).await();
+
+    // get a new unique opNamespace
+    logger.info("Assign unique namespace for Operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    logger.info("Assign unique namespace for Domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+
+    // Label the domain/operator namespace with istio-injection=enabled
+    Map<String, String> labelMap = new HashMap();
+    labelMap.put("istio-injection", "enabled");
+    assertDoesNotThrow(() -> addLabelsToNamespace(domainNamespace,labelMap));
+    assertDoesNotThrow(() -> addLabelsToNamespace(opNamespace,labelMap));
+
+    // install and verify operator
+    installAndVerifyOperator(opNamespace, domainNamespace);
+  }
+
+  /**
+   * Create a domain using model-in-image model.
+   * Add istio configuration with default readinessPort 
+   * Do not add any AdminService under AdminServer configuration
```

Comment:
why is this?

Reply:
This is the requirement for istio. No Network Channel